tests/unit/parser/test_float_literals.py

```
@@ -0,0 +1,150 @@
+
+    @pytest.mark.parametrize(
+        "expr,expected",
+        [
+            ("-1.0", -1.0),
+            ("-.5", -0.5),
+            ("-1e9", -1e9),
+            ("-.1e9", -1e8),
+            ("-1e-5", -1e-5),
+        ],
+    )
+    def test_negative_floats(self, parser, expr, expected):
```

Ruff (0.15.2)

[warning] Unused method argument: `expected`

(ARG002)